src/libraries/Microsoft.PowerFx.Core/Syntax/UserDefinitions.cs

```diff
@@ -65,15 +65,15 @@ public bool ProcessUserDefnitions(out UserDefinitionResult userDefinitionResult)
                 return false;
             }
                
-            var functions = CreateUserDefinedFunctions(parseResult.UDFs, out var errors);
+            var functions = CreateUserDefinedFunctions(parseResult.UDFs, out var errors, shouldBindBody);
 
             errors.AddRange(parseResult.Errors ?? Enumerable.Empty<TexlError>());
             userDefinitionResult = new UserDefinitionResult(functions, errors, parseResult.NamedFormulas);
 
             return true;
         }
 
-        private IEnumerable<UserDefinedFunction> CreateUserDefinedFunctions(IEnumerable<UDF> uDFs, out List<TexlError> errors)
+        private IEnumerable<UserDefinedFunction> CreateUserDefinedFunctions(IEnumerable<UDF> uDFs, out List<TexlError> errors, bool shouldBindBody = false)
```

Comment:
> = false [](http://example.com/codeflow?start=146&length=8)

This is a private method, and the only call is passing a value to this argument, so there's no need to define a default value for it.